content/kubermatic/master/tutorials-howtos/external-clusters/_index.en.md

```diff
@@ -1,55 +1,63 @@
  - The external Kubernetes cluster must be accessible using kubectl to get the information needed to add that cluster.
- - Make sure the cluster kubeconfig or provider credentials have sufficient rights to manage the cluster (get, list, upgrade,
- get kubeconfig)
+ - Make sure the cluster kubeconfig or provider credentials have sufficient rights to manage the cluster (get, list, upgrade,get kubeconfig)
 
-## Add External Cluster
+## Import External Cluster
 
-To add a new external cluster go to `Clusters` -> `External Clusters` and click the `Add External Cluster` button.
+To add a new external cluster go to `External Clusters` page and click the `Import External Cluster` button.
 
-![Add External Cluster](/img/kubermatic/master/tutorials/external_clusters/add_external_cluster.png "Add External Cluster")
+![Import External Cluster](/img/kubermatic/master/tutorials/external_clusters/add_external_cluster.png "Import External Cluster")
 
-Select the Kubernetes cloud provider. You can add the following external clusters:
 
-  - [GKE]({{< ref "./gke" >}})
-  - [AKS]({{< ref "./aks" >}})
-  - [EKS]({{< ref "./eks" >}})
+KKP allows connecting any existing Kubernetes cluster as an external cluster to view the cluster's current state. 
+
+To connect a cluster from any provider, click on `Any Provider` and provide the cluster name and kubeconfig. 
 
-## Connect Existing Cluster
+It is important that the kubeconfig used to connect the cluster is using standard authentication mechanisms like certificates or ServiceAccount tokens. OIDC or provider-specific plugins are not supported.
 
-KKP allows connecting any existing Kubernetes cluster as external cluster to view the cluster's current status. To connect a cluster from any provider, click on `Any Provider` and provide the cluster name and kubeconfig. It is important that the kubeconfig used to connect the cluster is using standard authentication mechanisms like certificates or ServiceAccount tokens. OIDC or provider-specific plugins are not supported.
+![Connect Cluster](/img/kubermatic/master/tutorials/external_clusters/connect.png "Connect Cluster")
 
 {{% notice info %}}
 If an existing kubeconfig uses custom authentication mechanisms, `kubermatic-installer convert-kubeconfig` can (optionally) be used to create a ServiceAccount on the external cluster and fetch its token into a new kubeconfig.
 {{% /notice %}}
 
-![Connect Cluster](/img/kubermatic/master/tutorials/external_clusters/connect.png "Connect Cluster")
-
 ![Provide kubeconfig](/img/kubermatic/master/tutorials/external_clusters/custom_cluster_credentials.png "Provide kubeconfig")
 
 You can then see the details of the cluster.
 
-![Custom Cluster](/img/kubermatic/master/tutorials/external_clusters/custom_details.png "Custom Cluster")
+![Custom Cluster](/img/kubermatic/master/tutorials/external_clusters/bringyourown.png "BringYourOwn Cluster")
+
+Select the Kubernetes cloud provider. You can add or create the following external clusters:
```

Comment:
This part here looks lost and isolated. To follow the documentation flow, IMO, this (line 54 to 58) should be moved to before (line 35) and the first lined be changed to:
```
After clicking the `Import External Cluster` button, select the Kubernetes cloud provider. You can add or create the following external clusters:
```